src/zenml/integrations/deepchecks/steps/deepchecks_data_drift.py

```diff
@@ -0,0 +1,108 @@
+
+import pandas as pd
+from deepchecks.core.suite import SuiteResult
+from pydantic import Field
+
+from zenml.integrations.deepchecks.data_validators.deepchecks_data_validator import (
+    DeepchecksDataValidator,
+)
+from zenml.integrations.deepchecks.validation_checks import (
+    DeepchecksDataDriftCheck,
+)
+from zenml.steps.base_step import BaseStep
+from zenml.steps.base_step_config import BaseStepConfig
+from zenml.steps.utils import clone_step
+
+
+class DeepchecksDataDriftCheckStepConfig(BaseStepConfig):
+    """Config class for the Deepchecks data drift validator step.
+
+    Attributes:
+        check_list: Optional list of DeepchecksDataDriftCheck identifiers
+            specifying the subset of Deepchecks data drift checks to be
+            performed. If not supplied, the entire set of data drift checks will
+            be performed.
+        dataset_kwargs: Additional keyword arguments to be passed to the
+            Deepchecks `tabular.Dataset` or `vision.VisionData` constructor.
+        check_kwargs: Additional keyword arguments to be passed to the
+            Deepchecks check object constructors. Arguments are grouped for
+            each check and indexed using the full check class name or
+            check enum value as dictionary keys.
+        run_kwargs: Additional keyword arguments to be passed to the
+            Deepchecks Suite `run` method.
+    """
+
+    check_list: Optional[Sequence[DeepchecksDataDriftCheck]] = None
+    dataset_kwargs: Dict[str, Any] = Field(default_factory=dict)
+    check_kwargs: Dict[str, Dict[str, Any]] = Field(default_factory=dict)
+    run_kwargs: Dict[str, Any] = Field(default_factory=dict)
+
+
+class DeepchecksDataDriftCheckStep(BaseStep):
+    """Deepchecks data drift validator step."""
+
+    def entrypoint(  # type: ignore[override]
+        self,
+        reference_dataset: pd.DataFrame,
+        target_dataset: pd.DataFrame,
+        config: DeepchecksDataDriftCheckStepConfig,
+    ) -> SuiteResult:
+        """Main entrypoint for the Deepchecks data drift validator step.
+
+        Args:
+            reference_dataset: Reference dataset for the data drift check.
+            target_dataset: Target dataset to be used for the data drift check.
+            config: the configuration for the step
+
+        Returns:
+            A Deepchecks suite result with the validation results.
+        """
+        data_validator = cast(
+            DeepchecksDataValidator,
+            DeepchecksDataValidator.get_active_data_validator(),
+        )
+
+        return data_validator.data_validation(
+            dataset=reference_dataset,
+            comparison_dataset=target_dataset,
+            check_list=cast(Optional[Sequence[str]], config.check_list),
+            dataset_kwargs=config.dataset_kwargs,
+            check_kwargs=config.check_kwargs,
+            run_kwargs=config.run_kwargs,
+        )
+
+
+def deepchecks_data_drift_check_step(
```

Comment:
I love these shortcut steps, btw. AFAIK they're only in integrations you've worked on. I'd love to see them everywhere where we have steps that take configs.

Reply:
Maybe worth adding as a ticket?

Reply:
done ! (the ticket)